Src/FluentAssertions/Types/TypeAssertions.cs

```diff
@@ -495,6 +484,22 @@ public AndConstraint<TypeAssertions> Implement<TInterface>(string because = "",
         return Implement(typeof(TInterface), because, becauseArgs);
     }
 
+    private bool SubjectImplements(Type interfaceType, string because = "", params object[] becauseArgs)
```

Comment:
🔧 We prefer to have a method always follow its invocations (like a book). So it should be moved below the last member that uses it.
🤔 Since this method actually throws under normal circumstances, I think naming it some like `AssertSubjectImplements`